### ISSUE TITLE
Invalid attribute name

### ISSUE THREAD

**Issue by msuMax** [paulw11/homebridge-wiser#4]
![default](https://user-images.githubusercontent.com/32818267/31586515-bcae65d2-b1da-11e7-8087-fd364c3de1c9.png)
Getting an error when connecting to Wiser and facing a '$' sign in a project.
I am guessing this problem may be related to xml2js library.

**Comment by arrikhan** [comment]
Have exact same error although different Lin/Column. Homebridge not accessible with this platform configured using example config exactly (except password/IP address per my setup).

**Comment by arrikhan** [comment]
> ![default](https://user-images.githubusercontent.com/32818267/31586515-bcae65d2-b1da-11e7-8087-fd364c3de1c9.png)
> Getting an error when connecting to Wiser and facing a '$' sign in a project.
> I am guessing this problem may be related to xml2js library.

Have you found a way around this?

**Comment by msuMax** [comment]
Nope, decided to switch to homebridge-cbus and bought second CNI. Happily using it for more than a year.

Best regards,
Maxim Pozdyshev

**Comment by arrikhan** [comment]
@paulw11 , is there a possibility to look into this? Is there any debugging I can do?

**Comment by beerygaz** [comment]
Hey @paulw11 (Wiko) long time no speak!  Inevitable that our C-Bus journey would coincide again at some stage.   I'm very new to homekit, so just starting out, but getting the same error mentioned here.  Could it be because I have multiple projects in Toolkit?

**Comment by paulw11** [comment]
It seems that the XML in the Wiser isn't valid; An XML attribute can't start with a $ sign.

I have added additional debug logging to the new version - Enable debug on your home bridge and then when it starts it will log the XML responses returned from the Wiser.  This will help identify the problem
